pkg/clients/clients.go

```diff
@@ -23,6 +23,47 @@ const (
 	dotSeparator = "."
 )
 
+const (
+	connected       = "connected"
+	disconnected    = "disconnected"
+	AllConnected    = "all_connected"
+	AllDisconnected = "all_disconnected"
+	PartConnected   = "partially_connected"
+)
+
+// State represents connection state.
+type State int
+
+const (
+	// Disconnected represents disabled connection.
+	Disconnected State = iota
+	// Connected represents enabled connection.
+	Connected
+)
+
+// String returns string representation of State.
+func (s State) String() string {
+	switch s {
+	case Disconnected:
+		return disconnected
+	case Connected:
+		return connected
+	default:
+		return Unknown
+	}
+}
```

Comment:
Please add `MarshalJSON` function for State 
In response I get like below : 
```json 
{
    "status": "all_connected",
    "connection_status": [
        {
            "channel_id": "cd866cf4-e671-4db9-b181-1e803ea756ba",
            "thing_id": "92a70298-8279-41f1-ae7e-0ecebe6b6f4a",
            "status": 1
        }
    ]
}
``` 
This is due to not avaialbe of `MarshalJSON` for state